io.elementary.mail.json

```diff
@@ -50,7 +53,7 @@
                 {
                     "type": "git",
                     "url": "https://gitlab.gnome.org/GNOME/evolution-data-server.git",
-                    "tag": "3.40.2"
+                    "branch": "3.46.4"
```

Comment:
I believe this is a tag, not a branch. Also, maybe it's better to keep the version here and in the OS in sync since we aren't really using the server from the sandbox.